src/shared/components/mutationTable/MutationTable.tsx

```diff
@@ -664,7 +667,16 @@ export default class MutationTable<
             download: ProteinChangeColumnFormatter.getTextValue,
             sortBy: (d: Mutation[]) =>
                 ProteinChangeColumnFormatter.getSortValue(d),
-            filter: ProteinChangeColumnFormatter.getFilterValue,
```

Comment:
```
Type '(data: Mutation[], filterString: string, filterStringUpper: string, indexedAnnotatedMutationsByGenomicLocation?: { [genomicLocation: string]: Mutation; } | undefined, isCanonicalTranscript?: boolean | undefined) => boolean' is not assignable to type '(data: Mutation[], filterString: string, filterStringUpper?: string | undefined, filterStringLower?: string | undefined) => boolean'.
  Types of parameters 'indexedAnnotatedMutationsByGenomicLocation' and 'filterStringLower' are incompatible.
    Type 'string | undefined' is not assignable to type '{ [genomicLocation: string]: Mutation; } | undefined'.
      Type 'string' is not assignable to type '{ [genomicLocation: string]: Mutation; } | undefined'.ts(2322)
```
I think tt's because in this method, I add fourth optional parameter `indexedAnnotatedMutationsByGenomicLocation`, while `filter` in LazyMobXTable has fourth optional parameter `filterStringLower`

Reply:
I see, makes sense.